matlab_simulation/05-estimation/kalman_filter/kf_example3.m

```diff
@@ -0,0 +1,105 @@
+% Kalman Filter Example 2: Multirate 2D Omnidirectional AUV example
```

Comment:
I don't remember what the error said. If you aren't able to recreate it I wouldn't worry about it. It could have been a MATLAB thing and not a coding thing.